index.js

```diff
@@ -2,25 +2,36 @@
 const { ProvidePlugin } = require("webpack")
 const filterObject = require("filter-obj")
 
-const excludeObjectKeys = (object, excludeKeys) => filterObject(object, key => !excludeKeys.includes(key))
+const filterObjectKeys = (includeAliases, excludeAliases) => {
```

Comment:
Function name is a bit too generic. I'd prefer something like `createAliasFilter`. Also, the options object could be provided to the function, and `includeAliases` and `excludeAliases` could be destructured from it.